rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatchTest.scala

```diff
@@ -65,52 +62,47 @@ class VarMatcherSpec extends FlatSpec with Matchers {
     val pattern: Send =
       Send(ChanVar(FreeVar(0)), List(EVar(wc), GInt(8)), false, BitSet(), true)
     val expectedResult = Some(Map[Int, Par](0 -> GPrivate("unforgeable")))
-    val result         = spatialMatch(target, pattern).runS(emptyMap)
-    result should be(expectedResult)
+    assert(spatialMatchOptionalTest(spatialMatch(target, pattern), expectedResult))
     val targetPar: Par  = target
     val patternPar: Par = pattern
-    val parResult       = spatialMatch(targetPar, patternPar).runS(emptyMap)
-    parResult should be(expectedResult)
+    assert(spatialMatchOptionalTest(spatialMatch(targetPar, patternPar), expectedResult))
   }
 
   "Matching a send's body" should "work" in {
     val target: Send =
       Send(Quote(GPrivate("unforgeable")), List(GInt(7), GInt(8)), false, BitSet())
     val pattern: Send  = Send(ChanVar(wc), List(EVar(FreeVar(0)), GInt(8)), false, BitSet(), true)
     val expectedResult = Some(Map[Int, Par](0 -> GInt(7)))
-    val result =
-      spatialMatch(target, pattern).runS(emptyMap)
-    result should be(expectedResult)
+    assert(spatialMatchOptionalTest(spatialMatch(target, pattern), expectedResult))
     val targetPar: Par  = target
     val patternPar: Par = pattern
-    val parResult       = spatialMatch(targetPar, patternPar).runS(emptyMap)
-    parResult should be(expectedResult)
+    assert(spatialMatchOptionalTest(spatialMatch(targetPar, patternPar), expectedResult))
   }
   "Matching a send" should "require arity matching in" in {
     val target: Send =
       Send(Quote(GPrivate("unforgeable")), List(GInt(7), GInt(8), GInt(9)), false, BitSet())
     val pattern: Send = Send(ChanVar(wc), List(EVar(FreeVar(0)), EVar(wc)), false, BitSet(), true)
-    val result        = spatialMatch(target, pattern).runS(emptyMap)
-    result should be(None)
+    assert(spatialMatchOptionalTest(spatialMatch(target, pattern), None))
   }
 
   "Matching extras with free variable" should "work" in {
     val target: Par  = GInt(7).prepend(GInt(8)).prepend(GInt(9))
     val pattern: Par = GInt(8).prepend(EVar(FreeVar(0)))
-    val result       = spatialMatch(target, pattern).runS(emptyMap)
-    result should be(Some(Map[Int, Par](0 -> GInt(9).prepend(GInt(7)))))
+    assert(
+      spatialMatchOptionalTest(spatialMatch(target, pattern),
+                               Some(Map[Int, Par](0 -> GInt(9).prepend(GInt(7))))))
   }
   "Matching extras with wildcard" should "work" in {
     val target: Par  = GInt(7).prepend(GInt(8)).prepend(GInt(9))
     val pattern: Par = GInt(8).prepend(EVar(Wildcard(Var.WildcardMsg())))
-    val result       = spatialMatch(target, pattern).runS(emptyMap)
-    result should be(Some(Map.empty[Int, Par]))
+    assert(spatialMatchOptionalTest(spatialMatch(target, pattern), Some(Map.empty[Int, Par])))
```

Comment:
maybe let's go a step further and make it:
```
assertSpatialMatch(target, pattern, Some(Map.empty[Int, Par]))
```

Reply:
Done.